app/_src/gateway/plugin-development/wasm.md

```diff
@@ -31,7 +31,7 @@ world`-type filter:
 
 ### Deploying the filter with Kong
 
-After writing and building the filter, the next steps are to configure Kong to enable Wasm support and add the filter, then associate the filter to a Route or a Service. You will then be able to start Kong and issue requests that go through the filter.
+After writing and building the filter, the next steps are to configure Kong to enable Wasm support and add the filter, then associate the filter to a route or a service. You will then be able to start Kong and issue requests that go through the filter.
```

Comment:
Is this correct? How so? Historically, we capitalize all Model entities in our documentation; this was even part of my initial review to @flrgh.

Reply:
@thibaultcha We lowercase all Kong object and entity names. https://docs.konghq.com/contributing/style-guide/#kong-specific-terms

Reply:
You will still see capitalized entities in the docs, especially older docs, but the standard is lowercase.

Reply:
Ok.